### PR TITLE
[App Service] `az webapp config set`: Add new parameter `--runtime` to allow users to update their stack by using single parameter

### PR DESCRIPTION
**Pull request by mstxsun**
**Related command**
`az webapp config set`

**Description**
Add new parameter `--runtime` for `az webapp config set` to allow users to update their stack by using single parameter

**Testing Guide**

`az webapp config set -g MyResourceGroup -n MyApp --runtime "java:17:TOMCAT:10.0"`
Update the stack of MyApp to Java 17 and Tomcat 10

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AppService

**Comment by mstxsun** [comment]
@jvano

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 1]
```diff
@@ -347,6 +347,8 @@ def load_arguments(self, _):
                        help="Number of pre-warmed instances a function app has")
             if scope == 'webapp':
                 c.ignore('reserved_instance_count')
+                c.argument('runtime', help="canonicalized web runtime in the format of Framework:Version, e.g. \"PHP:7.2\"."
```
Comment (yanzhudd) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:350]:
```suggestion
                c.argument('runtime', help="Canonicalized web runtime in the format of Framework:Version, e.g. \"PHP:7.2\"."
```

Comment (mstxsun) [reply src/azure-cli/azure/cli/command_modules/appservice/_params.py]: Resolved!

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**